src/app/garfile/craft/index.njk

```diff
@@ -25,7 +25,7 @@
 		{% include "common/templates/includes/errors.njk" %}
 	{% endif %}
 
-  <form id="page-form" action="/garfile/craft" autocomplete="off" class="form-group" method="post" role="form">
+  <form id="previous_aircraft" action="/garfile/craft" autocomplete="off" class="form-group" method="post" role="form">
```

Comment:
The "page-form" id is picked up by the disableAutoFill library, so changing the id is essentially the way to "disable" it.